weave/environment.py

```diff
@@ -144,11 +146,18 @@ def weave_server_url() -> str:
 
 
 def wandb_base_url() -> str:
-    pod_url = os.getenv("WANDB_PRIVATE_BASE_URL", "").rstrip("/")
-    return pod_url if pod_url != "" else wandb_frontend_base_url()
+    pod_url = os.getenv("WANDB_PRIVATE_BASE_URL")
+    if pod_url:
+        return pod_url.rstrip("/")
+
+    settings = Settings()
+    return os.environ.get("WANDB_BASE_URL", settings.base_url).rstrip("/")
 
 
 def wandb_frontend_base_url() -> str:
+    if wf_trace_server_url() == "http://127.0.01:6345":
```

Comment:
This might be true right now, but may not always be true - for example, what if you are running a local trace server against QA? i think we should just get in the habit of setting `WANDB_BASE_URL` if we want the printout to be correct.

Reply:
in fact, this raises yet a bigger point - what are all the URLs we need to have set? Can we list those somewhere centrally? Are there common "presets"? Seems like we at least have:
* where the frontend is served
* where the gorilla is located
* where the trace_server is located

Reply:
Can we add an ENV var: `WANDB_FRONTEND_URL`? 
Then:
- frontend: `WANDB_FRONTEND_URL`
- gorilla:`WANDB_BASE_URL`
- trace: `WF_TRACE_SERVER_URL`

@tssweeney